ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -90,10 +92,10 @@ class Endpoints(
       case req @ HttpRequest(GET, Uri.Path("/v1/packages"), _, _, _) =>
         (implicit lc => httpResponse(listPackages(req)))
       // format: off
```

Comment:
What happens if we just turn on formatting here?

Reply:
let's try out :boom:

Reply:
This:
```
      case req @ HttpRequest(
            GET,
            Uri(
              _,
              _,
              Slash(Segment("v1", Slash(Segment("packages", Slash(Segment(packageId, Empty)))))),
              _,
              _,
            ),
            _,
            _,
            _,
          ) => (
        implicit lc => downloadPackage(req, packageId)
      )
```
Ugly formatting intensifies in my opinion :joy: 
<img src="https://media.giphy.com/media/8D6YrBlWZJmnK/giphy.gif"/>